lib/apiary/command/publish.rb

```diff
@@ -58,11 +58,8 @@ def query_apiary
 
       return if source.nil?
 
-      begin
-        JSON.parse(source)
-        abort('Did you forget the --json flag?') unless @options.json
```

Comment:
Should we remove the option itself too?

Reply:
not needed imho , `--json` option works as it stated in the help
https://github.com/apiaryio/apiary-client/blob/master/lib/apiary/cli.rb#L40

Reply:
But isn’t it irrelevant then?

Reply:
maybe there is an usecase for that + bckw compatibility

Reply:
> error if you use unknown option

Oh, then we gotta be more careful next time

Reply:
= green ?